docs: module resolution for `next.config.ts` is currently limited to `CommonJS` (#71162)
vercel/next.js: docs/02-app/01-building-your-application/07-configuring/01-typescript.mdx

export default nextConfig
```

> > **Note**: Module resolutions for `next.config.ts` are currently limited to `CommonJS`.

Reviewer: ```suggestion
> **Note**: Module resolution in `next.config.ts` is currently limited to `CommonJS`.
```

> > **Note**: Module resolution in `next.config.ts` is currently limited to `CommonJS`. This may cause incompatibilities with ESM only packages being loaded in `next.config.ts`. 

Author: ```suggestion
> **Note**: Module resolution in `next.config.ts` is currently limited to `CommonJS`. This may cause incompatibilities with ESM only packages being loaded in `next.config.ts`.
```
